.travis.yml

```diff
@@ -9,8 +9,8 @@ cache:
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-        - php: 7.0
+        - php: '7.0'
```

Comment:
Version string should be quoted to not be parsed as number. We were just lucky...